src/taco-cli/cli/kit.ts

```diff
@@ -95,8 +104,9 @@ class Kit extends commands.TacoCommandBase {
         return Kit.promptUser(resources.getString("CommandKitSelectProjectUpdatePrompt"))
         .then(function (answer: string): Q.Promise<any> {
             if (answer && answer.length > 0 ) {
-                answer = answer.toLowerCase();
-                if (resources.getString("PromptResponseYes").split("\n").indexOf(answer) !== -1) {
+                // Try the lower-case version of the answer as well (This is needed so we can have better test coverage for "yes" and "no" response code paths)
```

Comment:
I don't follow this comment; why do we need to check both original-case and lower-case? why not just always convert to lower case as we did before?